ConwaysGameOfLife/Messages.cs

```diff
@@ -10,8 +10,8 @@ public class Messages
 
         public const string QuitGame = "Enter 'q' to quit Game.";
 
-        public const string Run = "Enter 'p' to play game.";
+        public const string Play = "enter 'p' to play game.";
 
-        public const string RequestLiveCell = "Please enter the x, y coordinates for one live cell (e.g. '0,1')."; 
+        public const string RequestLiveCell = "Please enter the x, y coordinates for one live cell (e.g. '0,1') "; 

```

Comment:
Here it doesn't matter so much given that it's just for a `const string`, but consistency is a good practice in code. Humans (especially software developers) are good at picking up patterns i.e. consistency. In this instance, there's a period at the end of all of the other `const string` values.

Speculating, but you may have removed it because it's close to the example? Perhaps removing the parentheses instead may make it less crowded?
```suggestion
        public const string RequestLiveCell = "Please enter the x, y coordinates for one live cell e.g. '0,1'."; 
```